Mathlib/Algebra/Ring/Ext.lean

```diff
@@ -0,0 +1,528 @@
+/-
+Copyright (c) 2024 Raghuram Sundararajan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Raghuram Sundararajan
+-/
+import Mathlib.Algebra.Ring.Defs
+import Mathlib.Algebra.Group.Ext
+-- import Mathlib
```

Comment:
A commented-out line that is no longer needed can be safely deleted (we can always check the commits to get it back).
```suggestion
```

Reply:
Must have forgotten to remove it long ago!